### PR TITLE
Update webpack to actually use titles

### PR DESCRIPTION
**Pull request by ryelo**
RHCLOUD-1417

### CODE REVIEW

File: config/base.webpack.plugins.js [hunk 1]
```diff
@@ -23,23 +23,23 @@ plugins.push(WriteFileWebpackPlugin);
  * @type {var}
  */
 const HtmlWebpackPlugin = new (require('html-webpack-plugin'))({
-    title: 'My App',
+    title: 'Home',
```
Comment (karelhala) [inline config/base.webpack.plugins.js]: I think that it should be `Red Hat Cloud services` am I right?

File: config/base.webpack.plugins.js [hunk 2]
```diff
@@ -23,23 +23,23 @@ plugins.push(WriteFileWebpackPlugin);
...
     filename: 'index.html',
     chunks: [ 'App', 'vendor' ],
     template: path.resolve(__dirname, '../src/index.html')
 });
 plugins.push(HtmlWebpackPlugin);
 
 const LogoutPage = new (require('html-webpack-plugin'))({
-    title: 'Logout',
+    title: 'Logged out',
```
Comment (karelhala) [inline config/base.webpack.plugins.js]: How about `RHCS | Logged out`

File: config/base.webpack.plugins.js [hunk 3]
```diff
@@ -23,23 +23,23 @@ plugins.push(WriteFileWebpackPlugin);
...
     filename: 'logout.html',
     chunks: [ 'Logout', 'vendor' ],
     template: path.resolve(__dirname, '../src/not-signed.html')
 });
 plugins.push(LogoutPage);
 
 const NotFound = new (require('html-webpack-plugin'))({
-    title: '404',
+    title: 'Not found',
```
Comment (karelhala) [inline config/base.webpack.plugins.js]: How about `RHCS | Not found`

File: config/base.webpack.plugins.js (hunk 2 above)
Comment (ryelo) [reply config/base.webpack.plugins.js]: As per above, we're not `RHCS`. We've also decided _not_ to use the pipes in the title as of https://github.com/RedHatInsights/landing-page-frontend/pull/34

This is tracked in RHCLOUD-1417

> 404 should say "Not found"
> Logout should say "Logged out"

File: config/base.webpack.plugins.js (hunk 3 above)
Comment (ryelo) [reply config/base.webpack.plugins.js]: Same as comment above.

File: config/base.webpack.plugins.js (hunk 1 above)
Comment (ryelo) [reply config/base.webpack.plugins.js]: We're not `Red Hat Cloud Services`. I've asked mrike about it, so this needs BU approval.

Comment (ryelo) [reply config/base.webpack.plugins.js]: @karelhala mrike says to use "Home"

**Comment by ryelo** [comment]
@karelhala going to re-request a review so you can ack my comments

**Pull request merged**